GitExtUtils/FileLogging/Logger.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GitExtUtils.FileLogging
+{
+    public static class Logger
```

Comment:
I very much detest static classes. They create an intense code coupling, affect scalability and seriously inhibit testability.

Reply:
Thats why I wanted early feedback. I share your opinion on static classes but I've found more than 130 static classes in the code base - so I thought it was an appreciated style. 

Logging is more of an "ambient dependency" in that we either access some static entity or we constructor-inject in *a lot of places*. I didn't want to spent hours injecting if you wanted a static class ;)